hadoop-ozone/ozone-manager/pom.xml

```diff
@@ -157,6 +157,13 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>jersey-client</artifactId>
     </dependency>
 
+    <!-- Overriding ranger-intg jackson version -->
+    <dependency>
+      <groupId>org.codehaus.jackson</groupId>
+      <artifactId>jackson-core-asl</artifactId>
+      <version>${jackson1.version}</version>
+    </dependency>
```

Comment:
This is what Ranger brings without the exclusion:

```
[INFO] |     +- org.codehaus.jackson:jackson-jaxrs:jar:1.9.13:compile
[INFO] |     |  +- org.codehaus.jackson:jackson-core-asl:jar:1.9.13:compile
[INFO] |     |  \- org.codehaus.jackson:jackson-mapper-asl:jar:1.9.13:compile
```

If we want to add explicit dependency, I think we should add `jackson-jaxrs`, not `jackson-core-asl`.

Reply:
Thanks @adoroszlai for looking at this. @swamirishi has added jaxrs in #4504 .

Reply:
Please check https://github.com/apache/ozone/pull/4504 for the fix.